python/cudf_polars/cudf_polars/dsl/expressions/boolean.py

```diff
@@ -81,6 +81,7 @@ def __init__(
         self.options = options
         self.name = name
         self.children = children
+        self.is_pointwise = all(c.is_pointwise for c in self.children)
```

Comment:
This needs to be `name`-dependent.

`All`, `Any`, `IsFirstDistinct`, `IsLastDistinct`, `IsUnique`, `IsDuplicated` are _not_ pointwise